src/Main.java

```diff
@@ -1,172 +1,100 @@
-        System.out.println(d);
-        long e = 987_678_965_549L;
-        System.out.println(e);
-        float f = 2.786f;
-        System.out.println(f);
-        double g = 27.12;
-        System.out.println(g);
+
+        int age = 8;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен " + age + " То он совершеннолетний");
+        } else {
+            System.out.println("Возраст совершеннолетия ещё не наступил, нужно немного подождать");
+        }
+
+
+        //Задача №2
+        int temperature = 6;
+        if (temperature < 5) {
+            System.out.println("На улице " + temperature + " градуса, нужно надеть шапку");
+        } else {
+            System.out.println("На улице " + temperature + " градусов можно идти без шапки");
+        }
+
 
         //Задача №3
-        byte studentsLP = 23;
-        System.out.println("Учеников у Людмилы Павловны" + studentsLP);
-        byte studentsAS = 27;
-        System.out.println("Учеников у Анны Сергеевны" + studentsAS);
-        byte studentsEA = 30;
-        System.out.println("Учеников у Екатерины Андреевны" + studentsEA);
-        short paper = 480;
-        System.out.println(paper + "Листов бумаги закупили для учеников");
-        int allStudents = studentsAS + studentsEA + studentsLP;
-        System.out.println("Общее количество учеников из трёх классов" + allStudents);
-        int numberOfSheets = paper / allStudents;
-        System.out.println("На каждого ученика рассчитано" + numberOfSheets + "листов бумаги");
+        int speed = 60;
+        if (speed > 60) {
+            System.out.println("Если скорость " + speed + " то придётся заплатить штраф");
+        } else {
+            System.out.println("Если скорость меньше " + speed + " можно ездить спокойно");
+        }
 
         //Задача №4
-        byte bottle = 16;
-        System.out.println("Бутылок производится за 2 минуты" + bottle);
-        short bottle2 = (short) (bottle / 2);
-        System.out.println("Бутылок производится за 1 минуту" + bottle2);
-        short bottle3 = (short) (bottle2 * 20);
-        System.out.println("За 20 минут машина произвела" + bottle3 + " штук бутылок");
-        short minute1day = 24 * 60;
-        System.out.println("В одном дне" + minute1day + "минут");
-        short bottle1Day = (short) (bottle2 * minute1day);
-        System.out.println("За 1 день машина произвела" + bottle1Day + "штук бутылок");
-        short minute3day = (24 * 3) * 60;
-        System.out.println("В 3-ёх днях" + minute3day + "минут");
-        short bottle3Day = (short) (bottle2 * minute3day);
-        System.out.println("За 3 дня машина произвела" + bottle3Day + "штук бутылок");
-        int minute1Month = 1440 * 30;
-        System.out.println("В одном месяце" + minute1Month + "минут");
-        int bottle1Month = minute1Month * bottle2;
-        System.out.println("За 1 месяц машина произвела" + bottle1Month + "штук бутылок");
+        int agePeople = 25;
```

Comment:
можно было не создавать новую переменную, а просто использовать уже созданную
```suggestion
        age = 25;
```

Reply:
Так я назвал переменную другим именем, так как это другая задача , разве вся информация  тогда не смешается ?

Reply:
Ну представь, что у тебя таких задач будет похожих 20 штук и везде нужен возраст. Фантазии не хватит придумывать разные названия переменным. Проще просто использовать одну и ту же переменную. Вот если бы в одной и той же задачи нужны были 2 переменные для хранения разных возрастов, то тут уже придётся 2 создавать

Reply:
А про смешивание информации - у тебя задачи разделены комментариями, поэтому всё будет визуально видно